### ISSUE TITLE
Compatibility Program tasks for the spec project team

### ISSUE THREAD

**Issue by fdesbiens** [eclipse-sparkplug/sparkplug#256]
Hi everyone.

The Sparkplug working group's website now features [instructions for organizations willing to get their compatible products certified](https://sparkplug.eclipse.org/compatibility/get-listed). There are a few tasks the project team needs to perform to make the process operational. Thus, I would be grateful if you could:

- Determine the URL to download to access the TCK binaries (presumably on download.eclipse.org)
- Set up a GitHub repository to handle the "Get Listed" requests
- Deploy the [issue template](https://docs.google.com/document/d/1NC0Mb8hkKSS8BC2GLjZWn_wacrrvyUvHBHKOU4-q5os) in the repository.

Once this is done, please let me know. I will update the website accordingly. I appreciate your support!

**Comment by wes-johnson** [comment]
The repo is now set up here: https://github.com/eclipse-sparkplug/sparkplug.listings

I am unable to create issue templates due to permissions - created an issue for that: https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/2260

**Comment by wes-johnson** [comment]
There is now a TCK README here: https://github.com/eclipse-sparkplug/sparkplug/blob/develop/tck/README.md

It points to a new issue template here: https://github.com/eclipse-sparkplug/sparkplug.listings/issues/new?assignees=&labels=getlisted%2Ctriage&template=GET-LISTED.yml&title=Get+Listed+Request%3A+%5BADD+DETAILS+HERE%5D

The only thing not complete in this issue is the download link for the TCK binaries. We are still wrapping up the Eclipse release process for the TCK. As soon as this is complete, we'll provide the download link

**Comment by wes-johnson** [comment]
Downloads are now available here: https://projects.eclipse.org/projects/iot.sparkplug/downloads
